src/gui/shell.h

```diff
@@ -230,6 +232,8 @@ private slots:
 	QPoint m_mouse_pos;
 	// 2/3/4 mouse click tracking
 	QTimer m_mouseclick_timer;
+	QLocale currentLocale;
```

Comment:
`m_` prefix for member variables.

We should also provide a sane default value:
QLocale m_currentLocale{ QLocale QLocale::system() };

nitpick: `QInputMethod* currentInputMethod`

If we don't need `QInputMethod` the signal/slot finishes, we should avoid caching the value. Less state, less bugs.

Reply:
m_prefix done, not sure if i understand, we need the ref to QInputMethod so that dynaic locale change can happen?!